scripts/deployment/arbitrum-local-template.json

```diff
@@ -9,21 +9,24 @@
             "name": "Arbitrum",
             "rpc": "http://localhost:8547",
             "chain_id": 412346,
-            "finality_period": 3600,
             "l1_messenger": ["ArbitrumL1Messenger", ${l1_messenger_args}],
             "l2_messenger": "ArbitrumL2Messenger",
+            "finality_period": 3600,
+            "transfer_cost": 0,
+            "target_weight_ppm": 0,
```

Comment:
Is there are reason why we set both `transfer_cost` and `target_weight_ppm` to zero?
I think nonzero values for these should work and would be more realistic (this is being used in our Arbitrum E2E test).

Reply:
The reason for it is, that our e2e tests only work with `0` fees. I used the transfer cost to nail down the fees to 0.
With the changes about the fee structure in the contract, I'm going to pin the `lpMarginPPM` to 0 so we can choose arbitrary values for the other parameters.
In general we could choose any value, but they influence when minFee or when lpFeePPM is applied, so we need to be careful whenever we apply fees (which isn't the case for e2e tests)

Reply:
regarding mainnet.json, I left it open on purpose, because we did this for all keys which are not clear yet, or have to be added upon deployment. Let me know if you think otherwise. But I'd like to let the mainnet.json fail if we did not modify the necessary values